### PR TITLE
feat(alpha-6): add `Katla` network configs

### PR DESCRIPTION
**Pull request by davidtaikocha**

### CODE REVIEW

**Comment by d1onys1us** [comment]
@davidtaikocha is this ready for review? LGTM

**Comment by davidtaikocha** [comment]
> @davidtaikocha is this ready for review? LGTM

ok will update some tags later and merge the PR

**Comment by davidtaikocha** [comment]
It's ready for review now @d1onys1us @cyberhorsey @2manslkh

File: docker-compose.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ x-logging: &logging
 
 services:
   l2_execution_engine:
-    image: gcr.io/evmchain/taiko-geth:jolnir
+    image: gcr.dockerproxy.com/evmchain/taiko-geth:katla
```
Comment (wolfderechter) [inline docker-compose.yml:10]: Is this dockerproxy really secure though? Can't find much info about it and seems like an easy way to introduce risks, maybe better to suggest as an alternative to people that can't connect to gcr.io

**Review by mask-pp — APPROVED** [review]

**Review by d1onys1us — APPROVED** [review]

**Pull request merged**